request/graphql/schema/types/scalars.go

```diff
@@ -0,0 +1,63 @@
+// Copyright 2023 Democratized Data Foundation
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package types
+
+import (
+	"encoding/hex"
+
+	"github.com/sourcenetwork/graphql-go"
+	"github.com/sourcenetwork/graphql-go/language/ast"
+)
+
+var BytesScalarType = graphql.NewScalar(graphql.ScalarConfig{
+	Name:        "Bytes",
+	Description: "The `Bytes` scalar type represents an array of bytes.",
```

Comment:
question:  If it is an array of bytes, why does it only appear to accept string values?

todo: I'm not sure if it does only accept string values, could you add an integration test documenting the beheviour when provided an array of numbers:
```
testUtils.CreateDoc{
	Doc: `{
		"name": "John",
		"data": [0,1,2,3...]
	}`,
},
```

Reply:
I believe this is a bug in the`client.Document` API. You can also do this to a string field:

```
testUtils.CreateDoc{
		Doc: `{
			"name":  12345
		}`,
}
testUtils.CreateDoc{
		Doc: `{
			"name":  false
		}`,
}
```

Reply:
Not quite a bug, we just dont do any checking of the saved value.  The read should fail though I think.  Does the read of `[0,1,2,3...]` fail?  Can we please document the behaviour, as it is a more likely question than assigning `1234` to a boolean :)